treeherder/etl/artifact.py

```diff
@@ -42,7 +45,13 @@ def store_text_log_summary_artifact(job, text_log_summary_artifact):
         ]:
             # classify job as `new failure` - for filtering, etc.
             job.failure_classification_id = 6
-            job.save()
+            job.save(update_fields=["failure_classification_id"])
+            # for every log_errors (TLE object) there is a corresponding bugs/suggestion
+            for tle in log_errors:
+                if tle.line_number == suggestion["line_number"]:
+                    tle.new_failure = True
+                    tle.save(update_fields=["new_failure"])
```

Comment:
using update_fields is a perf win, so I get a little bit of a win here for the existing code :)